features/step_definitions/invalid_form.rb

```diff
@@ -0,0 +1,23 @@
+# Empty form
```

Comment:
It is important the appointment of the class, in this case, it is not quite descriptive because it does not do reference to any specific form. I suggest you move it to the step that it belong. for example it do a relation at the sing up.
 this will be right if the steps ware transversales to all app.